bitbots_body_behavior/src/bitbots_body_behavior/actions/kick_ball.py

```diff
@@ -11,6 +11,26 @@ def perform(self, reevaluate=False):
             self.blackboard.animation.play_animation(self.right_kick)
 
 
+class KickBallRight(AbstractActionElement):
+    def __init__(self, blackboard, dsd, parameters=None):
+        super(KickBallRight, self).__init__(blackboard, dsd, parameters)
+        self.right_kick = 'kick_right'  # TODO get actual name of parameter from some config
```

Comment:
This config is currently in `bitbots_body_behavior/config/animations.yaml` but should probably be moved to another place or merged with other animation configuration files, for example the one in the HCM.

Reply:
I know. I just copied that part from the KickBall class because I thought that using an external configuration file for animations everywhere (what I'd support) wouldn't fit into this PR. It seems to be a different project than improving the kick decision.